### PR TITLE
Webinfra update tour info

### PR DESCRIPTION
**Pull request by n8kim1**
Update tour info: we now have 2 sprint tours! Also update the format info (to match battlecode, not battlehack).
Update the submission timer, too.

image

### CODE REVIEW

**Comment by n8kim1** [comment]
image

image

File: frontend/src/views/tournaments.js [hunk 1]
```diff
@@ -94,10 +94,17 @@ class Tournaments extends Component {
                                 </div>
                                 <div className="content">
                                     <p>
-                                        In response to competitor feedback, we have changed the format of the competition.
-                                        Teams are split into four divisions. Round Robin tournaments are held within each of these divisions.
-                                        From each division, the four teams with the highest win-ratio move on the the next round.
-                                        These top 16 will face eachother in another round-robin tournament.
+                                        Scrimmage rankings will be used to determine seeds for the Sprint and Seeding Tournaments. For all other tournaments, results from the previous tournament will be used to seed teams (where ties will be broken by the scrimmage ranking right before the tournament).
```
Comment (j-mao) [inline frontend/src/views/tournaments.js:97]: rename tourney

File: frontend/src/views/tournaments.js [hunk 2]
```diff
@@ -94,10 +94,17 @@ class Tournaments extends Component {
...
+                                    </p>
+                                    <p>
+                                        Tournaments will be in a <a href='https://en.wikipedia.org/wiki/Double-elimination_tournament'>double elimination</a> format, with the exception of both Sprint Tournaments, which are single elimination. The Final Tournament will start with a blank slate (any losses from the Qualifying Tournament are reset).
+                                    </p>
+                                    <p>
+                                        Even if you miss earlier tournaments, you can participate in later tournaments (except the Final Tournament).
+                                        This includes the Qualifying Tournament — you can participate even if you miss every other tournament (but your seed will be worse than the seed of everyone who participated in the Seeding Tournament).
```
Comment (j-mao) [inline frontend/src/views/tournaments.js:104]: no seeding, use scrim rank

File: frontend/src/views/tournaments.js [hunk 3]
```diff
@@ -94,10 +94,17 @@ class Tournaments extends Component {
...
+                                    </p>
+                                    <p>
+                                        Each match within a tournament will consist of 3 games on 3 different maps, and the team that wins the most games will advance.
```
Comment (j-mao) [inline frontend/src/views/tournaments.js:107]: *at least* 3 games on different maps

**Review by j-mao — CHANGES_REQUESTED** [review]

**Review by j-mao — APPROVED** [review]

**Pull request merged**